docs/components/plugins/APIBox.tsx

```diff
@@ -0,0 +1,21 @@
+import { PlatformName } from '../../types/common';
+
+import { H3 } from '~/components/plugins/Headings';
+import { PlatformTags } from '~/components/plugins/PlatformTag';
+import { STYLES_APIBOX } from '~/components/plugins/api/APISectionUtils';
+
+type APIBoxProps = {
+  header?: string;
+  platforms?: PlatformName[];
+  children: JSX.Element[];
+};
```

Comment:
```suggestion
type APIBoxProps = React.PropsWithChildren<{
  header?: string;
  platforms?: PlatformName[];
}>;
```

Let's use `PropsWithChildren` instead of specifying `children` manually.